packages/react-core/src/demos/JumpLinks.md

```diff
@@ -130,3 +130,11 @@ ScrollspyH2 = () => {
   );
 };
 ```
+
+
+### With drawer
+
+This demo shows how jump links can be used in combination with a drawer.
```

Comment:
Could you also call out what you used as the `scrollableSelector` in this case? Just for clarity since I believe that was the major question which lead to the development of these demos